From bc9748979521be50d8766c4d7cf2aae49b22748a Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 19 Oct 2023 20:08:43 +0200
Subject: [PATCH] default-dock-positions.json: Use variant for LuneOS

LuneOS wants to use different layout in Luna-Next-Cardshell for phone and tablet.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
 files/conf/default-dock-positions.json | 30 ++++++++++++++++++--------
 1 file changed, 21 insertions(+), 9 deletions(-)

diff --git a/files/conf/default-dock-positions.json b/files/conf/default-dock-positions.json
index 813241b..b4e6bf4 100644
--- a/files/conf/default-dock-positions.json
+++ b/files/conf/default-dock-positions.json
@@ -1,9 +1,21 @@
-{
-    "quicklauncher": [
-        "com.palm.app.browser_default",
-        "com.palm.app.email_default",
-        "com.palm.app.calendar_default",
-        "com.palm.app.messaging_default",
-        "com.palm.app.photos_default" 
-    ]
-}
+[
+	{
+		"type": "tablet",
+		"items": [
+				"com.webos.app.enactbrowser_default",
+				"com.palm.app.email_default",
+				"com.palm.app.calendar_default",
+				"org.webosports.app.messaging_default",
+				"org.webosports.app.memos_default" 
+			]
+	},
+	{
+		"type": "phone",
+		"items": [
+			"org.webosports.app.phone_default",
+			"org.webosports.app.messaging_default",
+			"com.palm.app.email_default",
+			"com.palm.app.calendar_default"
+		]
+	}
+]
